_projects/libremesh-internet-control.md

```diff
@@ -0,0 +1,61 @@
+---
+name: "LibreMesh Internet Control"
+desc: "Develop a simple interface for LibreMesh to enable internet control and service whitelisting, addressing the needs of communities."
+collaborating_projects:
+- LibreMesh
+- Intervoices
+developers_involved:
+- technician from the Guarani community
+- LibreMesh developers
+difficulty: high
+status: open
+initiatives:
+- GSoC 2024
+issues:
+- "Difficulty in tracking and whitelisting services like WhatsApp due to changing IPs"
```

Comment:
our thoughts behind this field were to add links to github issues or other issue trackers